canteen/templates/canteen/login.html

```diff
@@ -16,7 +18,9 @@
             <input class="Grake-button" type="submit" value="登录">
         </div>
     </form>
-    <p style="width: 250px; text-align: left; font-size: 0.6em; color: #455A64;">不妨尝试<a href="https://github.com/szes-maker/EAxT/releases/download/v0.2.1/org.szesmaker.ordermeal_v0.2.1.apk" target="_blank">食验客户端</a></p>
+    <div id='clientDownload' style='display:none;'>
+    <p>不妨尝试<a href="https://github.com/szes-maker/EAxT/releases/download/v0.2.1/org.szesmaker.ordermeal_v0.2.1.apk" target="_blank">食验客户端</a></p>
```

Comment:
It's OK to display the promotion here, to let users know we have an Android application.

Reply:
>Shall we add a simple Javascript to detect the user's browser? This kind of promotion is pointless on Apple devices.

Reply:
I admit that I did change my mind after posting that comment.
Anyway, advertisements are used to let people know something. I prefer to show it in the original form.

Reply:
What about alerting iOS users?
```
if(iOS) {
    var r=confirm("这是食验Android客户端，您确定要下载吗？");
    if(r)
        window.location='...'
}
```

Reply:
I think it's unnecessary. You can do it if you want.